src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -30,6 +44,7 @@ internal class InstallDialogViewModel : BindableBase
         private FileSelectionType _fileSelectionType;
         private bool _anyFileSelected;
         private bool _isTreeViewEmpty;
+        private const string _commaDelimiter = ",";
```

Comment:
>_commaDelimiter [](start = 29, length = 15)

nit: As this is only used in one location and doesn't change, I wouldn't have bothered with this

Reply:
Updated in version 10.

---
In reply to: [197934076](https://github.com/aspnet/LibraryManager/pull/246#discussion_r197934076) [](ancestors = 197934076)